charts/plex/templates/deployment.yaml

```diff
@@ -266,6 +270,10 @@ spec:
 {{- else }}
           claimName: "extradata-{{ .name }}"
 {{- end }}
```

Comment:
We are indeed missing an extra `{{- end }}` here.  There needs to be one to close the `if..` block and another to close the `range..` block.